modules/nodes-nodes-jobs-creating.adoc

```diff
@@ -34,12 +34,22 @@ spec:
         command: ["perl",  "-Mbignum=bpi", "-wle", "print bpi(2000)"]
       restartPolicy: OnFailure    <6>
 ----
-1. Optional value for how many pod replicas a job should run in parallel; defaults to `completions`.
-2. Optional value for how many successful pod completions are needed to mark a job completed; defaults to one.
-3. Optional value for the maximum duration the job can run.
-4. Option value to set the number of retries for a job. This field defaults to six.
-5. Template for the pod the controller creates.
-6. The restart policy of the pod. This does not apply to the job controller.
+1. Optionally, specify how many pod replicas a job should run in parallel; defaults to `1`.
+* For non-parallel jobs, leave unset. When unset, defaults to 1.
```

Comment:
s/1/`1` in both instances

s/job/Job - probably would help to do a find and replace for consistency